Automattic/jetpack
# Search: Add better upsell page

Fixes #23570 

#### Changes proposed in this Pull Request:
The PR proposes a upsell page when site is connected but has no search subscription. The page has so much in common with the connection page, and will combine them in the future.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Navigate to a site fully connected but has no search subscription
- Open `/wp-admin/admin.php?page=jetpack-search`
- Ensure the upsell page is displayed
- Ensure you could purchase search products thru the link and then return to Search Dashboard

![image](https://user-images.githubusercontent.com/1425433/163089255-b7f2f78f-f270-4083-84f9-cb217f4bc613.png)
